xtable-core/src/test/java/org/apache/xtable/hudi/TestHudiPartitionValuesExtractor.java

```diff
@@ -237,9 +237,9 @@ public void testDateAndTimeFormats(
                 .build());
 
     Map<String, String> pathToPartitionFieldFormat = new HashMap<>();
-    pathToPartitionFieldFormat.put(column.getSourceField().getPath(), format);
+    pathToPartitionFieldFormat.put(column.getSourceField().getName(), format);
```

Comment:
Let's keep this as `path`, it is required for Hudi which can use a nested field for a partition